Chatto/Source/ChatController/BaseChatViewController.swift

```diff
@@ -186,7 +186,15 @@ open class BaseChatViewController: UIViewController, UICollectionViewDataSource,
             // If we have been pushed on nav controller and hidesBottomBarWhenPushed = true, then ignore bottomLayoutMargin
             // because it has incorrect value when we actually have a bottom bar (tabbar)
 
-            if self.hidesBottomBarWhenPushed && (navigationController?.viewControllers.count ?? 0) > 1 && navigationController?.viewControllers.last == self {
+            // Also if instance of BaseChatViewController is added as childViewController to another view controller, we had to check all this stuf on parent instance instead of self
```

Comment:
@KaterinaPetrova Thank you for your changes!

May I ask you to move this code to a separate function `setupInputContainerBottomConstraint`? This method becomes too big now

Also, can you replace `Ctrl` with `Controller`. We are trying to avoid shorthand writing in this repo